.github/workflows/ci.yml

```diff
@@ -0,0 +1,296 @@
+name: Tests master
+
+on:
+  # This avoids having duplicate builds for a pull request
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  ############################################################################
+  # Lint jobs
+  ############################################################################
+  lint-py:
+    name: Lint Python
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false 
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.8'
+          architecture: 'x64'
+      - name: Install python dependencies
+        run: python -m pip install --editable .[test]
+      - name: Show python environment
+        run: |
+          python --version
+          python -m pip list
+      - name: Run python lint
+        run: pylint lektor
+      - name: Cache pip
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/setup.py') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
```

Comment:
I think these go before the installation of dependencies, otherwise the cache won't be restored before it's used.

Reply:
Fixed